woocommerce/woocommerce-gutenberg-products-block
# Product Gallery: Remove additional wrapper

This PR removes the unnecessary, additional wrapper from the Product Gallery block.

## What

Fixes #11012

## Why

After disabling the `remove_div_wrapper()` function which was breaking the Product Gallery block, there was an unnecessary wrapper present.

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. With a block theme, open `Appearance > Editor`.
2. Open `Templates > Single Product`.
3. Select `Convert to Blocks` if you are using the Classic Templates.
4. Add the `Product Gallery` block.
5. Go to the frontend and using an Inspector notice there's only one div with the `wp-block-woocommerce-product-gallery` class.


## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Add suggested changelog entry here.
